aad/azure-spring-boot-starter-active-directory/aad-resource-server/README.md

```diff
@@ -10,30 +10,38 @@ urlFragment: azure-spring-boot-sample-active-directory-resource-server
 # Azure OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server client library for Java
 
 ## Key concepts
-This sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts only.
+This sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts(client application) only.
 
 1. Obtain the access token from the HTTP request header.
-2. Use `JwtDecoder` to parse the access token into `Jwt`.
+2. Use [JwtDecoder](https://jwt.ms/) to parse the access token into `Jwt`.
```

Comment:
No. Here `JwtDecoder` is a java class name, not a URL.

Reply:
yes, updated.